packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -578,21 +499,23 @@ async function createRedwoodApp() {
   }
 
   const newAppDir = path.resolve(process.cwd(), targetDir)
-  const templateDir = path.resolve(__dirname, '../template')
+  const templatesDir = path.resolve(__dirname, '../templates')
 
   // Engine check
-  await executeCompatibilityCheck(templateDir, yarnInstall)
+  await executeCompatibilityCheck(path.join(templatesDir, 'ts'), yarnInstall)
```

Comment:
The engines will never be different between the TS and JS templates, but since the TS one is the source of truth, we use it here for the engines check.